101-powerbi-workspace-create/azuredeploy.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "powerbiWorkspaceName": {
+      "type": "string",      
+      "minLength": 3,
+      "maxLength": 63,
+      "metadata": {
+        "description": "the name of the PowerBI workspace collection name. It should match with the following regular expression: ^(?:[a-zA-Z0-9]+-?)+$ or it will raise an error. "
```

Comment:
If this is globally unique, let's mention that in the desc

Reply:
@bmoore-msft Thank you for reviewing the pull request. updated the description.